src/style/styled.d.ts

```diff
@@ -49,5 +49,19 @@ declare module 'styled-components' {
       valid: string
       highlightGradient: string
     }
+    badge: {
+      font: {
+        plus: string
```

Comment:
I did not use the bright new colors from Figma, because personally, I find the "pastel'-er version of the green and red more pleasing. However, I trust your opinion more! We can easily change the colors in the theme settings.

Reply:
Fine by me, I was still iterating. I'll do a final tweaking pass at the very end 🎨